packages/terra-status-view/src/StatusView.jsx

```diff
@@ -11,22 +13,27 @@ const StatusViewVariants = {
   NOMATCHINGRESULTS: 'no-matching-results',
   NOTAUTHORIZED: 'not-authorized',
   ERROR: 'error',
-  CUSTOM: 'custom',
 };
 
 const propTypes = {
   /**
-   * An array of actionable buttons to display.
+   * An array of objects containg terra-button properties. Use these to render buttons in the bottom section.
+   * Example:`[{ text: 'Button 1', key: 1, size: 'medium', variant: 'action', onClick: onClickFunction}]`
    */
-  children: PropTypes.node,
+  buttonAttrs: PropTypes.arrayOf(PropTypes.object),
 
   /**
-   * The glyph that is to be displayed for the `custom` variant.
+   * Display a custom glyph. Overrides a variant's default glyph.
    */
   customGlyph: PropTypes.node,
 
   /**
-   *  Determines if the content should be aligned vertically at the top of the container rather than in the middle.
+   * @private The intl object to be injected for translations. Provided by the injectIntl function.
+   */
+  intl: intlShape.isRequired,
```

Comment:
Might need an intl expert's opinion here but I don't think these need to be defined within the props anymore. I think it can be grabbed using the context.

Reply:
Weren't we moving away from context based on React API instability?

Reply:
I'm not entirely sure, we should get one of the intl people to clarify which way we should be implementing this.

Reply:
react-intl isn't using the _new_ Context API (yet), but it's still using context. To Stephen's point, both solutions are grabbing the `intl` object from the context. However, in the previous implementation, the StatusView is interfacing with context directly, whereas now `injectIntl` does the context interfacing and forwards `intl` to StatusView as a prop. The new way is future-proofed a bit, as `injectIntl` abstracts the Context API used so StatusView doesn't have to care. If we stick with the old way, StatusView would break when react-intl updates.

I say all this with the caveat that when react-intl does start using the new Context API, we could simplify the implementation here by using the useContext hook. We'd want to make changes eventually to remove that injectIntl wrapper because it's unnecessary at that point. However, we'd have some runway to do so without immediately failing.

TLDR: This is the best way to currently get access to `intl`.